packages/number/README.md

```diff
@@ -0,0 +1,32 @@
+# Number
+
+A collection of utilities to propery localize numerical values in WooCommerce
+
+## Installation
+
+Install the module
+
+```bash
+npm install @woocommerce/number --save
+```
+
+_This package assumes that your code will run in an **ES2015+** environment. If you're using an environment that has limited or no support for ES2015+ such as lower versions of IE then using [core-js](https://github.com/zloirock/core-js) or [@babel/polyfill](https://babeljs.io/docs/en/next/babel-polyfill) will add support for these methods. Learn more about it in [Babel docs](https://babeljs.io/docs/en/next/caveats)._
+
+## Usage
+
+```JS
+import { formatNumber, formatValue, calculateDelta } from '@woocommerce/number';
+
+// Formats a number using site's current locale.
+// Defaults to en-US localization
+const localizedNumber = formatNumber( 1337 ); // '1,377'
+
+// formatValue's first argument is a type: average, currency, or number
+// The second argument is the number/value to format
+const formattedAverage = formatValue( 'average', '10.5' ); // 11 just uses Math.round
+const formattedCurrency = formatValue( 'currency', '10.5' ); // $10.50 calls @woocommerce.formatCurrency
```

Comment:
Just a note that this example needs removed (and the comment above on line 24 needs updated) per https://github.com/woocommerce/wc-admin/pull/1399#discussion_r251542253